patientIdentifierType.md

```diff
@@ -0,0 +1,112 @@
+    ```
+
+* #### List patientIdentifierType by UUID.
+
+    Retrieve a patientIdentifierType by its UUID. Returns a `404 Not Found` status if patientIdentifierType does not exist in the system. If the user is not logged in to perform this action, a `401 Unauthorized` status is returned.
+
+    ```console
+    GET /patientidentifiertype/:target_patientIdentifierType_uuid
+    ```
+### Create a patientIdentifierType
+
+* To create a patientIdentifierType you need to specify the below properties in the request. If you are not logged in to perform this action, a `401 Unauthorized` status is returned.
+
+    #### Properties
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | string | label for the identifier
+    *description* | string | a small description about the patientIdentifier
+    *format* | string | label for the identifier
+    *formatDescription* | string | a small description about the patientIdentifier
+    *required* | boolean | a true/false value
+    *checkDigit* | boolean | a true/false value
+    *validator* | string | `org.openmrs.patient.IdentifierValidator`
+    *locationBehavior* | "REQUIRED" or "NOT USED" | behavior of the location with respect to the identifier 
+    *uniquenessBehavior* | string | label for the identifier
+
+    ```console
+    POST /patientidentifiertype
+    {
+        "name": "string",
+        "description": "string",
+        "format": "string",
+        "formatDescription": "string",
+        "required": true,
+        "checkDigit": true,
+        "validator": "string",
+        "locationBehavior": "REQUIRED",
+        "uniquenessBehavior": "string"
+    }
+    ```
+### Update a patientIdentifierType
+
+* Update a target patientIdentifierType with given UUID, this method only modifies properties in the request. 
+Returns a `404 Not Found` status if patientIdentifierType not exists. If user not logged in to perform this action, a `401 Unauthorized status returned`.
+
+    #### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *uuid* | `target_patientIdentifierType_uuid` | Target patientIdentifierType resource UUID
+
+    #### Properties
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *resource* | `PatientIdentifierType` | Patient resource with updated properties (same body as while creating the resource body)
+
+    ```console
+    POST /patientidentifertype/:target_patientidentifiertype_uuid
+    -d modified_patientidentifiertype_object
+    ```
+
+### Delete a patientIdentifierType
+
+* Delete or retire a target patientIdentifierType by its UUID. Returns a `404 Not Found` status if patientIdentifierType not exists. If user is not logged in to perform this action, a `401 Unauthorized` status returned.
+
+    #### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    **uuid** | `String` | uuid to delete
+    *purge* | `Boolean` | The resource will be voided/retired unless purge = 'true'
```

Comment:
Patient identifier types are metadata, which means they would be retired (marked as no longer valid for future data), not voided.